pages/instructorAssessmentStatistics/instructorAssessmentStatistics.js

```diff
@@ -26,65 +24,54 @@ const setFilenames = function (locals) {
   locals.statsByDateCsvFilename = prefix + 'scores_by_date.csv';
 };
 
-router.get('/', function (req, res, next) {
-  debug('GET /');
-  setFilenames(res.locals);
-  async.series(
-    [
-      function (callback) {
-        debug('query assessment_stats');
-        var params = { assessment_id: res.locals.assessment.id };
-        sqldb.queryOneRow(sql.assessment_stats, params, function (err, result) {
-          if (ERR(err, callback)) return;
-          res.locals.assessment_stat = result.rows[0];
-          callback(null);
-        });
-      },
-      function (callback) {
-        debug('query assessment_duration_stats');
-        // FIXME: change to assessment_instance_duration_stats and show all instances
```

Comment:
Is this `FIXME` indeed no longer relevant?

Reply:
I think that it's not really clear whether we actually want to see all instances rather than just the highest scoring instance per student (which is what we do currently). I added a comment (not a FIXME) about this in afebb922cd7eb2df8faf424e5cb1540032d806b0